lib/auth/webauthnwin/api.go

```diff
@@ -162,15 +162,25 @@ func Register(
 }
 
 var (
-	// PromptPlatformMessage is the message shown before Touch ID prompts.
-	PromptPlatformMessage = "Using platform authenticator, follow the OS dialogs"
+	// PromptPlatformRegisterMessage is the message shown before windows dialog prompts.
+	PromptPlatformRegisterMessage = "Using platform authenticator, follow the OS dialogs" +
+		" and select *new* device"
+	// PromptPlatformLoginMessage is the message shown before windows dialog prompts.
+	PromptPlatformLoginMessage = "Using platform authenticator, follow the OS dialogs" +
+		" and select *registered* device"
 	// PromptWriter is the writer used for prompt messages.
 	PromptWriter io.Writer = os.Stderr
 )
 
-func promptPlatform() {
-	if PromptPlatformMessage != "" {
-		fmt.Fprintln(PromptWriter, PromptPlatformMessage)
+func promptPlatformRegister() {
+	if PromptPlatformRegisterMessage != "" {
```

Comment:
I know this is from before the PR - but do you know if there's a reason why we hold these as variables rather than constants and do a check they aren't empty?

Reply:
that way you can disable prompt messages during testing, I think it was done due to that fact and I just copied it from other place but @codingllama can have more insights

Reply:
The idea here is that external packages can tweak/disable the messages, which is something I think we should take advantage of here.